data/editor/globals/controller.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace Elementor\Data\Editor\Globals;
+
+use Elementor\Data\Base\Controller as Controller_Base;
+
+class Controller extends Controller_Base {
+	public function get_name() {
+		return 'globals';
+	}
+
+	public function register_endpoints() {
+		$this->register_endpoint( '\Elementor\Data\Editor\Globals\Endpoints\Colors' );
```

Comment:
Todo fix register_endpoint
```suggestion
		$this->register_endpoint( 'Colors' ); // TODO: Add auto folder scanning.
```